extension/package.json

```diff
@@ -42,7 +42,7 @@
             "./server/com.microsoft.java.test.plugin-0.6.0.jar"
         ],
         "views": {
-            "explorer": [
+            "test": [
```

Comment:
Do we need update the VSCode engine version for this?

Reply:
@yaohaizh Yes good point. Will update that.